### PR TITLE
Sponsors 2018

### PR DESCRIPTION
**Pull request by terencehuynh**

### CODE REVIEW

File: _data/events/melbourne.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+sponsors:
+- tier: Platnium Sponsor
+  type: col-md-4
+  companies: [telstra]
+
+- tier: Gold Sponsors
+  type: col-md-3 col-6
+  companies: [accenture, atlassian, bosch, imc, infosys, wade, xero]
+  missing: [australia-post, wisetech]
```
Comment (terencehuynh) [inline _data/events/melbourne.yaml]: Note: `missing` key is more internal notation about what sponsors aren't featured on our site and images need to be collected.

File: _includes/event/melbourne/prizes.html [hunk 2]
```diff
@@ -50,95 +41,50 @@ <h4>
...
-        </div>
-      </div>
-      <div class="prize col-md-6">
+      <div class="prize col-md-4">
         <div class="prize-card">
           <div class="prize-card-inner">
-            <h4>Best Industrial IoT Hack <span>Sponsored by Bosch</span></h4>
-            <p class="brief">Awarded to the best IoT hack that can be used to increase output or efficiency in industries (e.g. smart agriculture, smart cities, etc).</p>
-            <p>A heated Bosch jacket for each member</p>
+            <h4>Best IoT/Hardware Hack</h4>
```
[the remark this answers is not in the document]
Comment (terencehuynh) [reply _includes/event/melbourne/prizes.html]: I can make it first on the same row.

**Comment by terencehuynh** [comment]
@erfanio The old sponsors are still being used in the sponsor us page. I've updated to use the new sponsors images instead of the old ones.

**Comment by sph-huynh** [comment]
Forwarded you JS logos this morning

**Comment by terencehuynh** [comment]
@sph-huynh Jane Street has just been added. Merging in now.

**Pull request merged**